components/ui/AppMegaDropdownMenu.vue

```diff
@@ -1,38 +1,63 @@
 <template>
   <article class="app-mega-dropdown">
-    <button
-      ref="button"
-      class="app-mega-dropdown__button"
-      :class="`app-mega-dropdown__button_${kind}`"
-      @click="toggleOpen"
+    <label
+      ref="searchFieldWrapper"
+      class="app-mega-dropdown__search-field-wrapper"
+      :class="`app-mega-dropdown__search-field-wrapper_${kind}`"
```

Comment:
We're mostly referring to the "search field" element as "filter", so we could go for a more consistent name:

```suggestion
      ref="filterWrapper"
      class="app-mega-dropdown__filter-wrapper"
      :class="`app-mega-dropdown__filter-wrapper_${kind}`"
```

Reply:
https://github.com/Qiskit/qiskit.org/pull/1823/commits/fcbab16c799ba652cb080ac1b2f00f17c59df0c1